test/README.md

```diff
@@ -90,6 +90,10 @@ You can set these environment variables to configure the test suite:
                    the Chrome Debug Protocol, on the given port. Don't use this
                    with parallel tests.
 
+In addition, you can also set the `cockpit.bots.images-data-dir` variable with
+`git config` to the location to store the (unprepared) virtual machine images.
+This takes precedence over `TEST_DATA`.
```

Comment:
For convenience, could this show a full command line, for easier copy&pasting?

Reply:
Sure.